ui/page/settings/view.jsx

```diff
@@ -225,6 +226,7 @@ class SettingsPage extends React.PureComponent<Props, State> {
           <div>
             <Card title={__('Language')} actions={<SettingLanguage />} />
             {/* @if TARGET='app' */}
+            <Card title={__('Wallet Server')} actions={<SettingWalletServer />} />
```

Comment:
I would move this to the bottom. Maybe this whole thing should just be inside of "experimental settings`?